continuous_integration/travis/install.sh

```diff
@@ -50,6 +52,14 @@ conda install -q -c conda-forge \
     sqlalchemy \
     toolz
 
+# Install pandas dev
+conda uninstall -y --force numpy pandas
+
+if [[ ${PANDAS_DEV} ]]; then
+    PRE_WHEELS="https://7933911d6844c6c53a7d-47bd50c35cd79bd838daf386af554a83.ssl.cf2.rackcdn.com"
+    pip install --pre --no-deps --upgrade --timeout=60 -f $PRE_WHEELS numpy pandas
```

Comment:
What version of numpy does this install? Master? If so I'd switch the flag from `PANDAS_DEV` to something more indicative that numpy is dev too.

Reply:
Ah, yes I meant to ask about that. It'll be NumPy dev as well. It doesn't need to be, but I figured we may as well.

Reply:
Numpy dev is fine, I'd just change the flag/comments.